src/main/scala/dpla/ingestion3/enrichments/TypeEnrichment.scala

```diff
@@ -0,0 +1,132 @@
+package dpla.ingestion3.enrichments
+import dpla.ingestion3.mappers.rdf.DCMIType
+import org.eclipse.rdf4j.model.IRI
+
+class TypeEnrichment extends VocabEnrichment[String] {
+
+  // Performs term lookup
+  private val typeLookup = new VocabLookup[String](
+    (term: String) => normalizationFunc(term)
+  )
+  // Normalizes retrieval and accessed terms
+  private def normalizationFunc(term: String): String = term.toLowerCase.trim
+
+  // DCMIType mappings
+  val dcmiType = DCMIType()
```

Comment:
I assume that these mappings come from the brain of GG?  For posterity, it might be nice to indicate the origin of the mappings so they can be maintained.

Reply:
👍 agree. Let me follow-up with Gretchen